store/tikv/region_cache.go

```diff
@@ -253,8 +256,8 @@ func (c *RegionCache) asyncCheckAndResolveLoop() {
 		case <-c.notifyCheckCh:
 			needCheckStores = needCheckStores[:0]
 			c.checkAndResolve(needCheckStores)
-		case addrs := <-c.notifyDieCh:
-			c.invalidStore(addrs)
+		case idleStore := <-c.notifyIdleCh:
```

Comment:
Can we ignore the idle notify?
The idle connections just being closed, doesn't mean the store is failed.